shardingsphere-features/shardingsphere-authority/shardingsphere-authority-common/src/main/java/org/apache/shardingsphere/authority/algorithm/natived/NativeAuthorityProviderAlgorithm.java

```diff
@@ -48,6 +48,6 @@ public void init(final Map<String, ShardingSphereMetaData> mataDataMap, final Co
     
     @Override
     public String getType() {
-        return "STORAGE";
+        return "NATIVE";
```

Comment:
why not enum ?

Reply:
The feature is provided by SPI which is dynamic.
Enum is static variable which user can not add new type with out change original enum.